### PR TITLE
Rails updated to 5.0

### PR DESCRIPTION
**Pull request by Danchek**

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 == README
 
+Contributors:
```
Comment (rakvium) [inline README.rdoc]: Should be a header (https://guides.github.com/features/mastering-markdown/)

File: README.rdoc [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
+-denis denisdotsenko7@gmail.com
```
Comment (rakvium) [inline README.rdoc]: For unordered lists please use `*`.

**Review by rakvium — CHANGES_REQUESTED** [review]

File: README.rdoc (hunk 1 above)
Comment (rakvium) [reply README.rdoc]: It's not a header yet.

Comment (rakvium) [reply README.rdoc]: It's not a header yet.

File: app/models/application_record.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  self.abstract_class = true
+end
```
Comment (rakvium) [inline app/models/application_record.rb]: Please add a newline to the end of the file.

File: README.rdoc (hunk 1 above)
Comment (Danchek) [reply README.rdoc]: Should be correct now

**Review by rakvium — APPROVED** [review]

**Pull request merged**